README.md

```diff
@@ -13,32 +13,51 @@ For a longer description and available features and options, see the
 
 # Installation
 
-Installing mkosi is easy, as it has no runtime Python dependencies (you will
-need all the tools to format filesystems and bootstrap the distribution
-appropriate for your image, though).
+You can install mkosi from your distribution with its package manager or the
+development version from git. The distribution packages are the preferred way to
+install mkosi.
```

Comment:
As long as this is the first paragraph, I'm happy. I don't even need to read the rest of the patch ;)))